docs/.env.example

```diff
@@ -0,0 +1,3 @@
+# Create one with no scope selected on https://github.com/settings/tokens/new
+# This token is used for fetching the repository releases.
+GITHUB_TOKEN=
```

Comment:
missing newline here?

Reply:
I would delete this file anyway, as it was only included in the bootstrapped docus project.